scripts/env.sh

```diff
@@ -40,7 +41,9 @@ OPENC3_PATH=$OPENC3_DIR/openc3.sh
     DNETWORK="docker network"
 #fi
 
-DBOX="ivvitc/nos3-64:20240430"
+# DBOX="ivvitc/nos3-64:20240618"
+DBOX="ivvitc/nos3-64:dev"
+# DBOX="ivvitc/nos3-64:fprime"
```

Comment:
Just a reminder to update this as we release a new container version

Reply:
noted, we should release container version first make sure it works then change and merge